src/res/translation/translation_fr_FR.ts

```diff
@@ -952,7 +952,7 @@
     <message>
         <location filename="../../clientdlgbase.ui" line="584"/>
         <source>Set up your audio, connect to a server and start jamming!</source>
-        <translation type="unfinished"></translation>
+        <translation>Configurez votre interface audio, connectez-vous à un serveur et commencez à bœuffer !</translation>
```

Comment:
Not sure about the blank before the exclamation mark -- might be common in French? In German it would be strange or even wrong.

Reply:
In french, some punctuation marks are preceded and / or followed by a space, others are not.
For the exclamation mark, the rule is : always a space before and after!
Another difficult rule of french langage!
I want to let this one as it is.

Reply:
>  I want to let this one as it is.

Yes, of course -- you are the specialist! Thanks for explaining.